tox.ini

```diff
@@ -1,9 +1,16 @@
 [tox]
 envlist =
-    py{27,34,35}-sphinx{13,14,15}
+    py{27,34,35,36}-sphinx{13,14,15}
```

Comment:
Yeah, 1.3 is old, but folks are still using it on RTD. I'd be curious to know stats around it. Anyways, it's easy enough to support both, the env state that it is failing on was moved (i thought in 1.5, but apparently 1.4)